packages/devtools_app/lib/src/screens/network/network_screen.dart

```diff
@@ -187,7 +180,7 @@ class _NetworkProfilerControlsState extends State<_NetworkProfilerControls>
   @override
   void initState() {
     super.initState();
-
+    addAutoDisposeListener(offlineDataController.showingOfflineData);
```

Comment:
you can remove this if you use OfflineAwareControls instead

Reply:
ok done